### ISSUE TITLE
The web URL is not working in OBS.

### ISSUE THREAD

**Issue by VideoFX** [t41372/Open-LLM-VTuber#44]
The web URL is not working in OBS.

To replicate:
- open OBS
- Make a new browser source
- Use URL http://localhost:12393

It works in Chrome OK, but not in OBS. How to stream using this?

**Comment by t41372** [comment]
Yeah I tried and it doesn't work. The OBS browser is weird. Don't use the OBS browser source for now.

**Comment by VideoFX** [comment]
> Yeah I tried and it doesn't work. The OBS browser is weird. Don't use the OBS browser source for now.

Thank you @t41372 for this amazing work, I am genuinely having fun with it. My favorite thing is the modularity. Very thoughtful design, and generous to share with us.
